0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>I think it might be worth briefly explaining what caching is. E.g., "... compile and cache, that is save the compiled version so that it doesn't need to be recompiled everytime we call it, a version ...". Perhaps also linking to Wikipedia (or perhaps some better resource) to give a longer description of caching. I just worry that many folks won't know the term.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>True. Added the decription and link to the first <code>jit</code> section.  </p>